components/frontend/src/utils.js

```diff
@@ -405,6 +400,15 @@ export function slugify(name) {
     return `#${name?.toLowerCase().replaceAll(" ", "-").replaceAll("(", "").replaceAll(")", "").replaceAll("/", "")}`
 }
 
+export function addCounts(object1, object2) {
+    // Assuming object1 and object2 are objects of the form {key1: count1, key2: count2, ...}, add them together
```

Comment:
Is it correct that we're assuming that both objects have the exact same set of keys? And values of 0 are never omitted for instance?

Reply:
Yeah, it is currently a correct assumption, but maybe you meant that the implementation of `addCounts` should check the assumption?

Reply:
I was wondering if it would be an idea to loop over the unique `set(object1.keys + object2.keys)`

Reply:
I think it's safer to just assert that both objects have the same keys. I'll add that.